### DISCUSSION TITLE
PCAP job's stuck at "INCOMPLETE"

### DISCUSSION THREAD

**Discussion by kornflake003-GITHUB** [Security-Onion-Solutions/securityonion#6392] [Unsupported Versions] [answered]
Hello,

I have a problem with PCAP module in SOC, in the alerts section, I observe activity on port (1433). But when trying to make a query with the PCAP module, it indicates me as an incomplete task. I attach some images of my configuration:

![Diapositiva1](https://user-images.githubusercontent.com/85261291/143979968-9b31505d-ddc9-41c9-8b45-4aefd9ff2573.PNG)
![Diapositiva2](https://user-images.githubusercontent.com/85261291/143979984-4babfe23-b1d5-458c-ad49-1f9ba0fbe7fe.PNG)
![Diapositiva3](https://user-images.githubusercontent.com/85261291/143979992-a811c348-9c7c-4f47-8478-051ec31b2102.PNG)
![Diapositiva4](https://user-images.githubusercontent.com/85261291/143980014-fcc29203-eede-45ed-a5b2-ecd8ae28d346.PNG)
![Diapositiva5](https://user-images.githubusercontent.com/85261291/143980025-fd33173e-dbeb-4ce1-b7fd-b18243c19977.PNG)

Thanks for your support.

@tomjones1977

**Comment by tomjones1977** [comment, marked as answer]
Looking at the screeshots above, you've requested a PCAP for 22 Nov, but the earliest PCAP stored on disk is 29 Nov.  If you have an alert that has fired within that timeframe, can you pivot to a PCAP from the Alerts tab?  That should set all the PCAP parameters for you and show you packets directly related to that snort rule firing.

**Reply by kornflake003-GITHUB** [reply]
!Thanks a lot! @tomjones1977 Now it´s working OK.
